.husky/pre-commit

```diff
@@ -2,3 +2,13 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 yarn lint
+
+# This block does prettier formatting and adds it to commit
```

Comment:
This makes prettier run after linting and adds the changes to the commit

Reply:
nice